api/query/cache/poll/poll.go

```diff
@@ -92,6 +97,7 @@ func wait(start time.Time, total time.Duration) {
 func StartMetadataPollingService(ctx context.Context, logger shared.Logger, interval time.Duration) {
 	logger.Infof("Starting Metadata polling service.")
 	toBeRemovedPRs := make([]string, 0)
+	// nolint:exhaustruct // TODO: Fix exhaustruct lint error.
```

Comment:
This one still needs to be added to the .yaml file and have this comment removed.